### DISCUSSION TITLE
DNS masq container CPU limit

### DISCUSSION THREAD

**Discussion by fabio-percivaldi** [mia-platform/community#109] [Q&A] [answered]
Hi Everybody,

we are constantly receiving CPUThrottlingHigh alarm on the API gateway, in particular on the DNS masq container:
```
33.01% throttling of CPU in namespace digital-platform-development for container dnsmasq in pod api-gateway
```
Usually to solve this problem internet suggest to increase the limit of the CPU; is it possible to increase the CPU limit of the dnsmasq container only? or it is a % of the api-gateway limit? or worst than that it's just an unmodifiable fixed value?

**Comment by davidebianchi** [comment, marked as answer]
Hi @fabio-percivaldi! It is a value without a UI to configure it at the moment.
It is possible to change it changing directly the `api-console-config.json` file, in `services['api-gateway'].additionalContainers[0].resources.cpuLimits.max`.

**Reply by fabio-percivaldi** [reply]
as far as it is not overriden by the console it's a good solution for us, thank you
